roles/ohpc_matlab/tasks/main.yaml

```diff
@@ -0,0 +1,9 @@
+- name: install the latest version of libXtst
+  yum:
+    name: libXtst
+    state: latest
+
+- name: Download matlab
+  get_url:
+    url: https://uab.box.com/v/matlab
```

Comment:
How was this URL generated?

Reply:
Custom URL shared link settings option on box

Reply:
Let's at least make this a variable in groups_var/all  so it can reasonably modified outside the playbook.